src/mpp/Constants.hpp

```diff
@@ -306,8 +306,7 @@ struct ExtValue { int8_t type; uint8_t offset; uint32_t size; };
 
 // The order of types must be exactly the same as in compact::Family!
 using Value_t = std::variant<
-	std::nullptr_t, bool, uint64_t, int64_t, float, double,
-	StrValue, BinValue, ArrValue, MapValue, ExtValue
->;
+	std::nullptr_t, std::monostate, bool, uint64_t, int64_t, float, double,
```

Comment:
What about nullopt here?. also I'm afraid of the comment.. I seems that we should not change this.

Reply:
This is actually dead code (used in previous codec version), so let's just drop it in a separate commit.